include/autoppl/expression/distribution/bernoulli.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <cassert>
+#include <random>
+#include <autoppl/util/var_expr_traits.hpp>
+#include <autoppl/util/dist_expr_traits.hpp>
+
+namespace ppl {
+namespace expr {
+
+template <typename p_type>
+struct Bernoulli
+{
+    static_assert(util::assert_is_var_expr_v<p_type>);
+
+    using value_t = util::disc_param_t;
+    using param_value_t = typename util::var_expr_traits<p_type>::value_t;
+    using dist_value_t = double;
+
+    Bernoulli(p_type p)
+        : p_{p} 
+    {}
```

Comment:
Took out assert - doesn't make sense to evaluate something when the whole point of building an expression is to compute lazily. This makes everything simpler too because it's consistent with the rest of the parts.